modules/common-attributes.adoc

```diff
@@ -39,3 +39,4 @@ endif::[]
 :mtc-full: Migration Toolkit for Containers
 :mtc-version: 1.4
 :mtc-version-z: 1.4.2
+:ocp: OpenShift Container Platform
```

Comment:
I think we'll need to check with @vikram-redhat on whether this should be "OpenShift Container Platform" or just "OpenShift". But we can always merge it in like this and adjust it later.

Reply:
@vikram-redhat can you confirm?

Reply:
Let's use "OpenShift Container Platform".